vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/selected_account_rule.rb

```diff
@@ -0,0 +1,15 @@
+module SecureRooms
+
+  module AccessRules
+
+    class SelectedAccountRule
+
+      def self.call(_user, _card_reader, _accounts, selected)
+        return :ok if selected.present?
```

Comment:
this ends up looking weird, since the existence of `selected` means it was found in the array returned by `accounts_for_product`